pdns/rec_channel.hh

```diff
@@ -94,7 +94,17 @@ struct StatsMapEntry {
   std::string d_prometheusName;
   std::string d_value;
 };
-typedef std::map<std::string, StatsMapEntry> StatsMap;
+
+class SimpleNaturalCompare
```

Comment:
The name seems a bit too generic for something that is 100% tied to a prefix dash number value.